{{cookiecutter.app_name}}/package.json

```diff
@@ -18,8 +18,11 @@
     "eslint": "7.29.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "3.4.0",
-    "prettier": "2.3.2",
     "postcss": "8.3.5",
+    "prettier": "2.3.2",
     "snowpack": "3.7.1"
+  },
+  "dependencies": {
+    {% if cookiecutter._bootstrap_addon == "yes" %}"bootstrap": "^5.0.2"{% endif %}
```

Comment:
Should we remove `^` to install the exact version?

Reply:
And should we move it to `dev-dependencies` as well? As Bootstrap's only used in development and build time, so I think we don't need to add it here

Reply:
Regarding `^` its a good question. I used to prefer have ^ (or at least ~) in order to have the latest version with compatibility. But I'm not sure what are the. conventions at Nimble :) 
(from Rails-template I can see it's a mix bag: sometimes no ^, sometimes with...)
@gutakk any preference on your side? :)

Reply:
At Nimble, we usually fix the version of the external packages because we can control the behavior and risk. We cannot rely on external packages as we don't know about backward compatibility if they update the new version. 😉